llvm/test/CodeGen/X86/uscmp.ll

```diff
@@ -0,0 +1,163 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
+; RUN: llc < %s -mtriple=x86_64-unknown-unknown -show-mc-encoding | FileCheck %s --check-prefixes=CHECK,NO-NDD
```

Comment:
```suggestion
; RUN: llc < %s -mtriple=x86_64-unknown-unknown | FileCheck %s
```

No need to check encoding in this test. You'll probably have to manually drop the NDD lines.